packages/tools/devtools/devtools-view/src/DevtoolsView.tsx

```diff
@@ -434,6 +434,13 @@ function Menu(props: MenuProps): React.ReactElement {
 	const menuSections: React.ReactElement[] = [];
 
 	menuSections.push(
+		<MenuSection header="Home" key="home-menu-section">
+			<MenuItem
+				isActive={currentSelection?.type === "homeMenuSelection"}
+				text="Home"
+				onClick={onHomeClicked}
+			/>
+		</MenuSection>,
```

Comment:
Let's try not to have an item inside the menu if it's not necessary. In the ADO item I suggested making it so MenuSection could be a Header-only thing. Maybe adding a prop to it for a `onHeaderClick` or similar.